### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by crazysparrow69**
Added dateOfCompletion field to tasks
Implemented statistics
Fixed docs by removing useless fileds
Added validations to some controllers
Documentation

### CODE REVIEW

File: README.md
Comment (LikerFeed) [inline README.md:1]: It's not very polite to steal docs from someone else's pool)

File: controllers/taskController.js
Comment (LikerFeed) [inline controllers/taskController.js:1]: good work!

**Review by LikerFeed — APPROVED** [review]
Good!

**Review by Dimdim28 — APPROVED** [review]
Amazing, now please create API for tasks sharing )

**Pull request merged**